### ISSUE TITLE
[1.14.4 HD U F4 pre1] Forge accesstransformers do not work for OptiFine classes

### ISSUE THREAD

**Issue by Exopandora** [sp614x/optifine#2737]
My mod ShoulderSurfing uses forge and has an accesstransformer for the following method

`public net.minecraft.client.renderer.ActiveRenderInfo func_216782_a(DDD)V`

Nevertheless the jvm crashes with an `java.lang.IllegalAccessError` when trying to call the method when OptiFine is installed. OptiFine overrides this class but the signature of the method remains the same (checked bytecode).

Using:
Minecraft Forge 1.14.4-28.0.45
OptiFine_1.14.4_HD_U_F4_pre1 (extracted)
ShoulderSurfing-1.18.1

Runtime dump of the methods (with accesstransformer and optifine):
```
public final net.minecraft.util.math.Vec3d net.minecraft.client.renderer.ActiveRenderInfo.func_216787_l()
public net.minecraft.util.math.BlockPos net.minecraft.client.renderer.ActiveRenderInfo.func_216780_d()
public net.minecraft.fluid.IFluidState net.minecraft.client.renderer.ActiveRenderInfo.func_216771_k()
public net.minecraft.entity.Entity net.minecraft.client.renderer.ActiveRenderInfo.func_216773_g()
public net.minecraft.util.math.Vec3d net.minecraft.client.renderer.ActiveRenderInfo.func_216785_c()
public float net.minecraft.client.renderer.ActiveRenderInfo.func_216778_f()
public float net.minecraft.client.renderer.ActiveRenderInfo.func_216777_e()
public void net.minecraft.client.renderer.ActiveRenderInfo.func_216783_a()
public void net.minecraft.client.renderer.ActiveRenderInfo.func_216772_a(net.minecraft.world.IBlockReader,net.minecraft.entity.Entity,boolean,boolean,float)
public void net.minecraft.client.renderer.ActiveRenderInfo.func_216781_o()
protected void net.minecraft.client.renderer.ActiveRenderInfo.func_216776_a(float,float)
protected void net.minecraft.client.renderer.ActiveRenderInfo.func_216775_b(double,double,double)
protected void net.minecraft.client.renderer.ActiveRenderInfo.func_216784_b()
protected void net.minecraft.client.renderer.ActiveRenderInfo.func_216782_a(double,double,double)
private double net.minecraft.client.renderer.ActiveRenderInfo.func_216779_a(double)
protected void net.minecraft.client.renderer.ActiveRenderInfo.func_216774_a(net.minecraft.util.math.Vec3d)
public boolean net.minecraft.client.renderer.ActiveRenderInfo.func_216770_i()
public final net.minecraft.util.math.Vec3d net.minecraft.client.renderer.ActiveRenderInfo.func_216788_m()
public net.minecraft.block.BlockState net.minecraft.client.renderer.ActiveRenderInfo.getBlockState()
public net.minecraft.block.BlockState net.minecraft.client.renderer.ActiveRenderInfo.getBlockAtCamera()
public boolean net.minecraft.client.renderer.ActiveRenderInfo.func_216786_h()
```

**Comment by Exopandora** [comment]
Accesstransformer still work for classes that have not been overriden by OptiFine

**Comment by sp614x** [comment]
OptiFine 1.14.4_HD_U_F4_pre3
 - fixed Forge access transformers (#2737)

**Comment by jakey1995abc** [comment]
It's not fixed for 1.12.2...
